logging_config.yaml

```
@@ -0,0 +1,37 @@
+# It contains the configuration for logging
+# You can find more info in https://docs.python.org/3/library/logging.config.html#logging-config-api
+version: 1
+formatters:
+  detailed:
+    format: "%(asctime)s - %(name)s - %(levelname)s - %(message)s"
+filters:
+  warnings_filter:
+    (): logging.Filter
+    name: "py.warnings"
+  info_only_filter:
+    (): GANDLF.utils.gandlf_logger.InfoOnlyFilter
+handlers:
+  stdoutHandler: # only display info level
+    class: logging.StreamHandler
+    level: INFO
+    formatter: detailed
+    filters: [info_only_filter]
+    stream: ext://sys.stdout
+  stderrHandler: # display warning and above messages
+    class: logging.StreamHandler
+    level: ERROR
```

```suggestion
    level: WARNING
```

Now stderr displays only errors, while warnings are hidden